### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by a0m0rajab** [caressofsteel/hugo-story#41]
it would be great if we had something to explain the classes and styles types with a visual type in the reference and adding them to the readme file.

**Comment by jwehrle** [comment]
I concur. For instance, I am using the content-align-left tag but on mobile it overrides and uses content-align-center. I can't figure out how to change this.

**Comment by caressofsteel** [comment]
@a0m0rajab @jwehrle I agree it would be great to have documentation. The implementation is pretty straight forward, so anyone with the time could dive in and document how things are wired up.

@jwehrle You may find your answer somewhere in the Sass side of the project.
